extensions/amp-story/0.1/amp-story-consent.js

```diff
@@ -178,20 +175,19 @@ export class AmpStoryConsent extends AMP.BaseElement {
     this.assertAndParseConfig_();
 
     const storyEl = closestByTag(this.element, 'AMP-STORY');
+    const consentEl = closestByTag(this.element, 'AMP-CONSENT');
     const logoSrc = storyEl && storyEl.getAttribute('publisher-logo-src');
 
     if (!logoSrc) {
       user().warn(
           TAG, 'Expected "publisher-logo-src" attribute on <amp-story>');
     }
 
-    const consentId = Object.keys(this.consentConfig_.consents)[0];
-
     // Story consent config is set by the `assertAndParseConfig_` method.
     if (this.storyConsentConfig_) {
       this.storyConsentEl_ = renderAsElement(
           this.win.document,
-          getTemplate(this.storyConsentConfig_, consentId, logoSrc));
+          getTemplate(this.storyConsentConfig_, consentEl.id, logoSrc));
```

Comment:
Is the presence of an id validated?  Or do we need to assert it here?

Reply:
Good question. I just checked, and it looks like [amp-consent](https://github.com/ampproject/amphtml/blob/master/extensions/amp-consent/0.1/amp-consent.js#L130) does it